autoload/leaderf/python/leaderf/fileExpl.py

```diff
@@ -564,13 +564,11 @@ def _getFilesFromCache(self):
             else:
                 return None
 
-    @removeDevIcons
     def setContent(self, content):
         self._content = content
         if lfEval("g:Lf_UseCache") == '1':
             self._writeCache(content)
 
-    @showDevIcons
     def getContent(self, *args, **kwargs):
         files = kwargs.get("arguments", {}).get("--file", [])
         if files:
```

Comment:
```python
        files = kwargs.get("arguments", {}).get("--file", [])
        if files:
            result = []
            for file in files:
                with lfOpen(file, 'r', errors='ignore') as f:
                    result += f.readlines()
            return result
```
these lines can be encapsulated as a function and decorate it with @showDevIcons.

Reply:
Good!
Changed 7fd8cd924d58523e6266dc43b15fb3af7443b475.